src/Console/Application.php

```diff
@@ -98,6 +103,14 @@ public function run(InputInterface $input = null, OutputInterface $output = null
         return parent::run($input, $output);
     }
 
+    private function isInitialTestPhpOptionHasXdebug(InputInterface $input): bool
+    {
+        return (bool) preg_match(
+            '/(zend_extension\s*=.*xdebug.*)/mi',
+            (string) $input->getParameterOption('--initial-tests-php-options', true)
```

Comment:
probably, the second parameter (default value) is not needed here as we have default value empty string in `configure()` method

Reply:
parameters binding is not happening there.
To avoid manual type cast (with `(string)`) we can replace boolean value to the empty string instead